.yarnrc.yml

```diff
@@ -1 +1 @@
-nodeLinker: node-modules
+nodeLinker: node-modules
```

Comment:
Can you add a new line?

Reply:
What?

Reply:
New line at the end of the file*
I should've been more specific

Reply:
Fixed